42_grml

```diff
@@ -74,10 +74,18 @@ get_dependencies() {
     break
   done
   echo $dependencies
-
 }
 
-
+get_iso_identifier()
+{
+  local iso=${1}
+  local id=
+  if isoinfo --version >/dev/null 2>&1 ; then
```

Comment:
Is there any specific reason why we invoke isoinfo with --version and not check for its availability/executable?
(No big deal though and fine for me, just curious!)

Reply:
I figure it's better than `test -x` (requires hard-coding the binary location) or trying to use `which`...